src/main/java/no/ssb/dlp/pseudo/service/sid/Redactor.java

```diff
@@ -0,0 +1,31 @@
+package no.ssb.dlp.pseudo.service.sid;
+
+import com.google.common.base.Strings;
+
+/**
+ * Redacts identifiers used by SID mapping.
+ */
+public class Redactor {
+
+    static int MAX_VISIBLE_CHARS_FNR = 6;
+    static int MAX_VISIBLE_CHARS_SNR = 4;
```

Comment:
I guess this will be safe enough.. alternatively only show the first 3 chars.

Reply:
Yes, maybe 3 chars should be enough. I can change it.